src/prompts/workflowInitPrompt.ts

```diff
@@ -75,6 +83,104 @@ function substituteVariables(template: string, variables: PromptVariables): stri
   return result;
 }
 
+/**
+ * Build branch auto-derivation instructions for the agent.
+ * 
+ * When the user skips providing a branch name, the agent needs instructions
+ * on how to derive an appropriate branch name from context.
+ * 
+ * @param issueUrl - Optional issue URL that can provide context for branch derivation
+ * @returns Markdown content with branch derivation instructions
+ */
+function buildBranchAutoDeriveSection(issueUrl: string | undefined): string {
```

Comment:
Similar to how workItemInitPrompt.template.md has substantial prompt context in a markdown file for easier editing, I would like the content of buildBranchAutoDeriveSection and buildWorkDescriptionSection to stored as template markdown files dynamically loaded.